src/plugins/console/public/application/containers/editor/monaco/utils/status_code_decoration_utils.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { monaco } from '@kbn/monaco';
+import { RequestResult } from '../../../../hooks/use_send_current_request/send_request';
+import {
+  DEFAULT_STATUS_BADGE_CLASSNAME,
+  SUCCESS_STATUS_BADGE_CLASSNAME,
+  PRIMARY_STATUS_BADGE_CLASSNAME,
+  WARNING_STATUS_BADGE_CLASSNAME,
+  DANGER_STATUS_BADGE_CLASSNAME,
+} from './constants';
+
+const getStatusCodeClassName = (statusCode: number) => {
```

Comment:
Using the same logic we used for the [Ace badge](https://github.com/elastic/kibana/blob/b51c50347994a984d5a46ca94b3f7f361bd0c5da/src/plugins/console/public/application/components/network_request_status_bar/network_request_status_bar.tsx#L33).